app/models/request.rb

```diff
@@ -83,4 +84,8 @@ def set_defaults
     self.number_of_children = 0
     self.number_of_finished_children = 0
   end
+
+  def policy_name
```

Comment:
Name implies string. Change to `policy_class`. We may be able to move this method to metadata.rb
```
def policy_class
  "#{model_name.name}Policy".contantize
end
```
```